### PR TITLE
Use mkdirSync for Lambda handshake file

### PR DESCRIPTION
**Pull request by willarmiros**
*Description of changes:*
* Adds Package lock to .gitignore
* Cleans up creation of file for handshaking in Lambda env 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 *.log
 docs
+package-lock.json
```
Comment (leepa) [inline .gitignore]: package-lock.json is supposed to be committed - what's the reason for removal?

File: packages/core/lib/env/aws_lambda.js [hunk 2]
```diff
@@ -24,14 +24,16 @@ module.exports.init = function init() {
...
-      var filename = '/tmp/.aws-xray/initialized';
-      fs.closeSync(fs.openSync(filename, 'a'));
-      var now = new Date();
-      fs.utimesSync(filename, now, now);
-    });
-  });
+  try {
+    fs.mkdirSync('/tmp/.aws-xray/');
+  } catch (e) {
+    logger.getLogger().debug('Lambda handshake directory already exists.');
```
Comment (leepa) [inline packages/core/lib/env/aws_lambda.js]: This catches all issues and not just if the directory exists so the log entry isn't accurate?

**Comment by sthulb** [comment]
This pull request is addressing the wrong thing. The code that creates `/tmp/.aws-xray/` should be removed – it's not read by any code at any point, no internal service or SDK is reading this file.

**Comment by willarmiros** [comment]
Closing in favor of #221

**Pull request closed without merging**